hoomd/mpcd/test-py/CMakeLists.txt

```
@@ -12,6 +12,7 @@ set(TEST_LIST
     integrate_integrator
     integrate_slit
     integrate_slit_pore
+    make_random_sphere
```

When you rename the file below, change this and move it up after `init_make_random`
```suggestion
    init_make_random_sphere
```